README.md

```diff
@@ -1,7 +1,231 @@
 # maps4all-signup
 [![Circle CI](https://circleci.com/gh/hack4impact/maps4all-signup.svg?style=svg)](https://circleci.com/gh/hack4impact/maps4all-signup)
 
-A signup page for maps4all to allow for one-click deployment of the [maps4all](https://github.com/hack4impact/maps4all) project 
+
 
 ## License
 [MIT License](LICENSE.md)
+
+
+## Team Members
+
+- Santiago Buenahora
+- Roberta Nin Feliz
+- Rani Iyer
+- Katie Jiang
+- Hunter Lightman
+- Hana Pearlman
+- Ben Sandler
+- Stephanie Shi
+- Abhinav Suri
+
+## Synopsis
+
+A signup page for maps4all to allow for one-click deployment of the [maps4all](https://github.com/hack4impact/maps4all) project 
+
+## Setting up
+
+##### Clone the repo
+
+```
+$ git clone https://github.com/hack4impact/maps4all-signup.git
+$ cd maps4all-signup
+```
+
+##### Initialize a virtualenv
+
+```
+$ pip install virtualenv
+$ virtualenv -p python3 venv
+$ source venv/bin/activate
+```
+(If you're on a mac) Make sure xcode tools are installed
+```
+$ xcode-select --install
+```
+
+##### Install the dependencies
+
+```
+$ pip install -r requirements.txt
+```
+
+##### Other dependencies for running locally
+
+You need to install [Foreman](https://ddollar.github.io/foreman/) and [Redis](http://redis.io/). Chances are, these commands will work:
+
+```
+$ gem install foreman
+```
+
+For Mac (using [homebrew](http://brew.sh/)):
+
+```
+$ brew install redis
+```
+
+For Linux (Fedora)
+
+```
+$ sudo dnf install redis
+```
+
+For Linux (Debian/Ubuntu):
+
+```
+$ sudo apt-get install redis-server
+```
+
+If you don't want to install redis locally, you can use Redis container with docker
+
+```
+$ docker pull redis:latest
+$ docker run -d -p 6379:6379 --name maps4all-redis redis:latest
+```
+
+##### Set your environment variables
+
+Create a `.env` file in your directory and include the following variables:
+
+* `MAIL_PASSWORD` and `MAIL_USERNAME` are your login credentials for [Sendgrid](https://sendgrid.com/).
+* `TWILIO_ACCOUNT_SID` and `TWILIO_AUTH_TOKEN` allow you to use the Twilio API to send text messages. They can be obtained through the [Twilio console](https://www.twilio.com/login).
+* `ADMIN_EMAIL` and `ADMIN_PASSWORD` allow you to login as an administrator to Maps4All-Signup on your local machine.
+* `HEROKU_CLIENT_ID` and `HEROKU_CLIENT_SECRET` allow you to host the new Maps4All instances on Heroku.
```

Comment:
We should include instructions on how to get these:

- Create an account on Heroku
- Use the command `heroku auth:login` to sign in to Heroku in terminal
- Run the command `heroku clients:create "maps4all" http://localhost:5000` to get the `HEROKU_CLIENT_ID` and `HEROKU_CLIENT_SECRET`